src/core/Akka.Streams.Tests/Dsl/StageActorRefSpec.cs

```diff
@@ -178,15 +178,15 @@ public async Task A_Graph_stage_ActorRef_must_ignore_and_log_warnings_for_Poison
             warn.Message.ToString()
                 .Should()
                 .MatchRegex(
-                    "<PoisonPill> message sent to StageActor\\(akka\\://AkkaSpec/user/StreamSupervisor-[0-9]+/\\$\\$[a-z]+\\) will be ignored, since it is not a real Actor. Use a custom message type to communicate with it instead.");
+                    "<PoisonPill> message sent to StageActor\\(akka\\://StageActorRefSpec-1/user/StreamSupervisor-[0-9]+/\\$\\$[a-z]+\\) will be ignored, since it is not a real Actor. Use a custom message type to communicate with it instead.");
```

Comment:
Need to change this because it was hardwired as "AkkaSpec" before the cleanup.

Reply:
Further changed by using string interpolation, injecting `ActorSystem.Name` instead. No more hard wiring.